com.unity.render-pipelines.high-definition/Documentation~/Custom-Pass-API-User-Manual.md

```diff
@@ -0,0 +1,67 @@
+# Custom Pass Utils API User Manual
+
+## Blur
+
+### Gaussian Blur
+
+The gaussian blur function will allow you to blur an image with an arbitrary radius and quality (number of samples). For performance reasons, you also have the possibility to run the blur kernel after a downsample pass.
+
+Here's an example of Custom Pass that blurs the camera color buffer:
+
+```CSharp
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+using UnityEngine.Rendering;
+using UnityEngine.Experimental.Rendering;
+
+class GaussianBlur : CustomPass
+{
+    RTHandle    halfResTarget;
+
+    protected override void Setup(ScriptableRenderContext renderContext, CommandBuffer cmd)
+    {
+        halfResTarget = RTHandles.Alloc(
+            // Note the * 0.5 here, it will allocate a half res target, which saves a lot of memory
+            Vector2.one * 0.5f, TextureXR.slices, dimension: TextureXR.dimension,
+            // We don't need alpha for this effect so we use an HDR no alpha texture format
```

Comment:
// Since alpha is unnecessary for Gaussian blur, this effect uses an HDR texture format with no alpha channel.